src/Components/Common/BedSelect.tsx

```diff
@@ -65,18 +69,18 @@ export const BedSelect = (props: BedSelectProps) => {
       selected={selected}
       onChange={setSelected}
       showNOptions={showNOptions}
-      placeholder="Search by beds name"
+      placeholder={t("Search by beds name")}
       fetchData={onBedSearch}
       optionLabel={(option: any) => {
         if (Object.keys(option).length === 0) return "";
         return (
-          `${option.name} ${option?.location_object?.name || "Unknown"}` ||
+          `${option.name} ${option?.location_object?.name || t("Unknown")}` ||
           option?.location_object?.name
         );
       }}
       optionLabelChip={(option: any) => {
         if (Object.keys(option).length === 0) return "";
-        return `${option?.bed_type}` || "Unknown";
+        return `${option?.bed_type.replace(/_/g, " ")}` || t("Unknown");
```

Comment:
@Bhavik-ag you should be adding the bed type keys and display strings to English locale files so that it's translated.